spec/cocoa/nsattributedstring_spec.rb

```diff
@@ -54,7 +54,50 @@
       "test\n  ".attrd.strip.should == 'test'.attrd
       "  \n test".attrd.strip.should == 'test'.attrd
     end
+  end
+
+  describe "should allow joining an array of attributed strings" do
+    it "should not care about standard arrays of strings" do
+      joined = ['a', 'b', 'c'].join_attrd
+      joined.should == 'abc'
+      joined.is_a?(NSAttributedString).should == false
+      joined.is_a?(NSString).should == true
```

Comment:
True, that's a better way to do that, but if someone said "does `join_attrd` return an `NSAttributeString`?" I'd rather answer "yes!" instead of "depends..."